pkg/gofr/logging/remotelogger/dynamic_level_logger_test.go

```diff
@@ -23,15 +23,16 @@ func TestRemoteLogger_UpdateLevel(t *testing.T) {
 	}))
 
 	rl := remoteLogger{
-		remoteURL:          mockServer.URL,
-		levelFetchInterval: 1,
+		remoteURL: mockServer.URL,
+		// TODO: use a time.Duration for levelFetchInterval
```

Comment:
should we fix this TODO in this PR itself?

Reply:
I think so, because this int in seconds implies that the tests cannot be shorter than 1s

Reply:
done